src/pykeen/losses.py

```diff
@@ -51,6 +51,7 @@
     Square Error                   $g(s, l) = \frac{1}{2}(s - l)^2$
     Binary Cross Entropy           $g(s, l) = -(l*\log (\sigma(s))+(1-l)*(\log (1-\sigma(s))))$
     Pointwise Hinge                $g(s, l) = \max(0, \lambda -\hat{l}*s)$
+    ??                             $g(s, l) = \log(1+\exp(\lambda-\hat{l}*s))$
```

Comment:
@mberr is there any added value to having a margin here? It is symmetric to the pairwise losses, that also has a non-zero margin with the softplus (i.e., soft margin). Could this be called the soft pointwise hinge loss?